app/views/trainees/outcome_details/recommended.html.erb

```diff
@@ -11,16 +11,23 @@
         <%= trainee_name(@trainee) %> recommended for <%= @trainee.award_type %>
       </h1>
     </div>
+    <h2 class="govuk-heading-m">Getting <%= @trainee.award_type %> status</h2>
     <p class="govuk-body">
       The Department for Education will award <%= @trainee.award_type %> where appropriate within 3
       working days.
     </p>
   </div>
 </div>
 
-<p class="govuk-body govuk-!-margin-bottom-9"><%= govuk_link_to("Check trainee statuses from your list of records", trainees_path) %></p>
-
 <%= render FeedbackLink::View.new(
   enabled: Settings.features.enable_feedback_link,
-  feedback_link_url: Settings.feedback_link_url,
+  feedback_link_url: Settings.recommended_for_award_feedback_url,
+  feedback_type_text: "recommending a trainee for #{@trainee.award_type}",
 ) %>
+
+<h2 class="govuk-heading-m">Next steps</h2>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li class="govuk-body govuk-!-margin-bottom-2"><%= govuk_link_to("view #{trainee_name(@trainee)}", trainee_path(@trainee)) %></li>
```

Comment:
This text is incorrect - it's intended to be in the form `view Foo Bar’s record`

Reply:
fixed 👍